go.mod

```diff
@@ -30,10 +30,17 @@ replace (
 	github.com/bketelsen/crypt => github.com/bketelsen/crypt v0.0.3
 	github.com/coreos/etcd v3.3.10+incompatible => github.com/coreos/etcd v3.3.25+incompatible
 	github.com/dgrijalva/jwt-go => github.com/dgrijalva/jwt-go/v4 v4.0.0-preview1
+	github.com/gin-gonic/gin v1.4.0 => github.com/gin-gonic/gin v1.8.1
```

Comment:
go dependencies upgrades should be notified in changelog. Just in cases it breaks something, it's easier to track.